go/teams/implicit.go

```diff
@@ -122,7 +134,7 @@ func lookupImplicitTeamAndConflicts(ctx context.Context, g *libkb.GlobalContext,
 			impTeamName.IsPublic, "could not find team with suffix: %v", preResolveDisplayName)
 	}
 	team, err := Load(ctx, g, keybase1.LoadTeamArg{
-		ID:          imp.TeamID,
+		ID:          teamID,
```

Comment:
this was the bug....

Reply:
Note that we set this field up at line 123, but we were previously ignoring it.  That might account for the behavior @strib sees in CI

Reply:
Oh! Good find